api/src/main/java/ai/djl/ndarray/BaseNDManager.java

```diff
@@ -72,6 +72,25 @@ public final Device defaultDevice() {
         return getEngine().defaultDevice();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void closeSubManagersWithoutResources() {
```

Comment:
These two class is not needed.

Reply:
removed two added methods from NDManager hierarchy that are not necessary